package.json

```diff
@@ -7,7 +7,7 @@
     "test": "jest",
     "start": "webpack --watch",
     "build": "webpack",
-    "setup": "npm i; npm run build; npm link; cd tc-ui-toolkit-test && npm i; npm link tc-ui-toolkit;"
+    "setup": "npm i && npm run build && npm link && cd tc-ui-toolkit-test && npm i && npm link tc-ui-toolkit"
```

Comment:
This was not working in Windows' Git Bash with semicolons, so used &&'s to make it work.